docs/developer/app-store/apps/stripe/webhooks.mdx

```diff
@@ -0,0 +1,164 @@
+}
+}
+```
+
+For failed operation it will set either `CHARGED_FAILURE` or `AUTHORIZATION_FAILURE` in Saleor and return data:
+
+```json
+{
+  "paymentIntent": {
+    "errors": [
+      {
+        "code": "error code",
+        "message": "human readable message"
+      }
+    ]
+  }
+}
+```
+
+Possible codes are:
+- `StripeCardError`
+- `StripeApiError`
+
+## `TransactionProcessSession`
+
+TransactionProcessSession is supposed to be executed once Stripe payment is completed on the storefront. This webhook will fetch Payment Intent from Stripe
+and update Saleor transaction with the last state resolved from Payment Intent status.
+
+This webhook doesn't require any data in input.
+
+It can set the following statuses on Saleor transaction (based on Stripe PaymentIntent status) in case of successful processing:
+
+- `CHARGE_SUCCESS`
+- `AUTHORIZATION_SUCCESS`
+- `CHARGE_ACTION_REQUIRED`
+- `AUTHORIZATION_ACTION_REQUIRED`
+- `CHARGE_REQUEST`
+- `AUTHORIZATION_REQUEST`
+- `CHARGE_FAILURE`
+- `AUTHORIZATION_FAILURE`
+
+In some cases it can fail to process and return the following data to Storefront
+
+```json
+{
+"paymentIntent": {
+  "errors": [{
+    "code": "error code",
+    "message": "human readable message"
+  }]
+  }
+}
+```
+
+If error occurs, it will set `CHARGE_FAILURE` or `AUTHORIZATION_FAILURE` on Saleor transaction.
+
+Possible codes are:
+- `StripeCardError`
+- `StripeApiError`
+
+
+## `TransactionChargeRequested`
+
+This webhook is executed when a staff user wants to charge the payment that was authorized. It will call Stripe API to capture Payment Intent and update Saleor transaction with `CHARGE_SUCCESS` status when capture is successful or `CHARGE_FAILURE` when there is an error from Stripe API.
+
+When Payment Intent is successfully captured Stripe will emit `payment_intent.succeeded` event which will be handled by app Stripe webhook.
+
+## `TransactionCancelationRequested`
+
+This webhook is executed when staff user wants to cancel the payment that was authorized or it is in pending state (have `CHARGE_ACTION_REQUIRED` or `AUTHORIZATION_ACTION_REQUIRED` status). It will call Stripe API to cancel Payment Intent and update Saleor transaction with `CANCEL_SUCCESS` status is cancelation is successful or `CANCEL_FAILURE` when there is an error from Stripe API.
+
+When Payment Intent is successfully canceled Stripe will emit `payment_intent.canceled` event which will be handled by app Stripe webhook.
+
+## `TransactionRefundRequested`
+
+This webhook is executed when a staff user wants to refund the payment that was charged. It will call Stripe API to create a refund and return Stripe refund id. Actual status of refund will be set in the app' Stripe webhook when handling `charge.refund.updated` event based Stripe refund status:
+
+* `succeeded` will report `REFUND_SUCCESS` status
+* `pending` or `requires_action` will report `REFUND_REQUEST` status
+* `failed` or `canceled` will report `REFUND_FAILURE` status
+
+
+# Stripe Webhook
```

Comment:
When using `#`, the section is missing in the right bar. Not sure if this is expected or not
![image](https://github.com/user-attachments/assets/345f22b4-2196-4498-827c-ccffaa46088d)

Reply:
Looks like docosaursus is using `#` as a root heading so in content we must do `##` and lower